lib/plugins/aws/info/display.test.js

```diff
@@ -67,6 +67,12 @@ describe('#display()', () => {
     expect(consoleLogStub.calledOnce).to.equal(true);
     expect(message).to.equal(expectedMessage);
 
+    awsInfo.options.conceal = true;
+    expectedMessage = `${chalk.yellow('api keys:')}`;
+    expectedMessage += '\n  keyOne';
+    const concealedMessage = awsInfo.displayApiKeys();
+    expect(concealedMessage).to.equal(expectedMessage);
+
```

Comment:
Could you please add new tests for running the command with `--conceal` instead of including it into the existing test case.